python/ray/experimental/client/__init__.py

```diff
@@ -1,44 +1,67 @@
-import ray
-from ray.experimental.client.worker import Worker
+from ray.experimental.client.api import ClientAPI
+from ray.experimental.client.api import APIImpl
 from typing import Optional
 
-_client_worker: Optional[Worker] = None
-_in_cluster: bool = True
+import logging
+
+logger = logging.getLogger(__name__)
+
+_client_api: Optional[APIImpl] = None
 
 
 def get(*args, **kwargs):
-    global _client_worker
-    global _in_cluster
-    if _in_cluster:
-        return ray.get(*args, **kwargs)
-    else:
-        return _client_worker.get(*args, **kwargs)
+    global _client_api
+    if _client_api is None:
+        from ray.experimental.client.core_ray_api import CoreRayAPI
```

Comment:
This assumes ray client and core ray API cannot be used at the same place right? Is it our plan?

Reply:
Yes, that's correct.